### PR TITLE
Mende

### PR DESCRIPTION
**Pull request by ghost**
Review please

### CODE REVIEW

File: src/Library.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ private Library() {
 	}
 
 	
-	public static synchronized Library INSTANCE() {
+	public static synchronized Library Instance() {
```
Comment (nhuthuynh) [inline src/Library.java]: getInstance will be better

**Review by nhuthuynh — CHANGES_REQUESTED** [review]
Change the instance method

File: src/Library.java (hunk 1 above)
Comment (ghost) [reply src/Library.java]: Done

**Comment by nhuthuynh** [comment]
All good

**Review by nhuthuynh — APPROVED** [review]

**Pull request merged**